config/jobs/kubernetes/sig-docs/sig-docs-periodics.yaml

```diff
@@ -0,0 +1,15 @@
+periodics:
+  - interval: 168h # 1 week
+    name: kubernetes-website-link-checker
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    spec:
+      containers:
+        - image: wjdp/htmltest
+          # TODO irvifa find a way to clone and run the following commands
```

Comment:
This page of the docs could be useful https://github.com/kubernetes/test-infra/blob/master/prow/pod-utilities.md

For cloning the k/website
```yaml
periodics:
- name: periodic-cluster-api-provider-aws-test-creds
  annotations:
    testgrid-dashboards: sig-docs-kubernetes-website
    testgrid-tab-name: website-link-check-result
  decorate: true  # pod utility decoration - this is the key thing to be added.
  interval: 4h
  labels:
    preset-service-account: "true"
    preset-k8s-ssh: "true"
  extra_refs:
  - org: kubernetes
    repo: website
    base_ref: master
    path_alias: "k8s.io/website"
    workdir: true  # This will make the cloned path the default working directory (may be useful)
  spec:
    containers:
    - image: wjdp/htmltest
      ...
```